### PR TITLE
config: parse bind interface

### PR DESCRIPTION
**Pull request by nirmoy**
## What does this PR change?
allow  parsing Network.Bind.Interface. Note: parsing ipv4 addr only for now
## Documentation
- No documentation needed

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 
 	kubicutil "github.com/kubic-project/kubic-init/pkg/util"
+	"github.com/vishvananda/netlink"
```
Comment (MalloZup) [inline pkg/config/config.go]: why aren't we using `net` std pkg?

File: pkg/config/config.go [hunk 2]
```diff
@@ -327,8 +328,21 @@ func (kubicCfg KubicInitConfiguration) IsSeeder() bool {
...
+		if err != nil {
+			return nil, err
+		}
+		for _, addr := range addrs {
+			// return 1st IPv4 address
+			if len(addr.IP) >= 4 {
+				return addr.IP, nil
+			}
+		}
+		return nil, fmt.Errorf("No address found in %s", kubicCfg.Network.Bind.Interface)
```
Comment (inercia) [inline pkg/config/config.go]: I think maybe we can avoid the use of `netlink` and stick to the standar library with something like this:

```suggestion
		ifaces, err := net.Interfaces()
		if err != nil {
		    return nil, err
		}

		for _, i := range ifaces {
		    if strings.ToLower(i.Name) != strings.ToLower(kubicCfg.Network.Bind.Interface) {
		        continue
		    }
    
		    addrs, err := i.Addrs()
		    if err != nil {
		        return nil, err
		    }

		    for _, addr := range addrs {
		        var ip net.IP
		        // just return the first IP (maybe we could do some smarter heuristics...)
		        switch v := addr.(type) {
		        case *net.IPNet:
		                return v.IP, nil
		        case *net.IPAddr:
		                return v.IP, nil
		        }
		    }
		}

		return nil, fmt.Errorf("No address found in %s", kubicCfg.Network.Bind.Interface)

```

File: pkg/config/config.go (hunk 1 above)
Comment (nirmoy) [reply pkg/config/config.go]: I was using that a lot, net std pkg make sense too as we are not doing any heavy lifting.

File: pkg/config/config.go (hunk 2 above)
Comment (inercia) [reply pkg/config/config.go]: Cool, much better!

**Pull request merged**